collections/_article/godot-at-gdc-2024.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Godot Does Conferences - Meet us at GDC!"
+excerpt: "We are excited to announce that the Godot Engine will attend the 2024 Game Developers Conference (March 18-22)."
+categories: ["events"]
+author: Godot Foundation
+image: /storage/blog/gdc/2024/gdc-thumbnail.webp
+date: 2024-02-13 12:00:00
+---
+
+Since we had a great & productive time last year, and 2.000 pins dispersed around the world to prove it, we are attending the [Game Developer Conference](https://gdconf.com/) (GDC) again, which is taking place March 18-22 in San Francisco, California. 
+
+## What can you expect? 
+
+This year, [W4 Games](https://w4games.com/) is organizing a Godot Pavilion and the Godot Foundation will be there hosting a selection of developers who could not be more stoked to let you play their Godot Engine games on the expo floor. Keep an eye on this space to not miss the line-up announcement! 
+
+We are also bringing along Godot Foundation members (like Adam, Emi, George, Nat, Remi, Winston & more) for in person chats, as well as inviting any maintainer/contributor present for work reasons to come join us any time :)
+
+**You will find us at: booth S763, North & South Halls, Moscone Center.**
+
+If you would like to schedule a meeting with the Godot Foundation, you can write an email to [Emilio](mailto:emilio@godotengine.org) with "GDC" in the subject.
+
+Other than that, we are also planning a Godot get-together at GitHub HQ on March 20th, more info on that as soon as we can share it.
+
+## Where can I stay up to date?
+
+This blog is the right place. In addition, we put out a forum post <link> asking the community who plans to attend, to give you the chance to connect and make plans together. Lastly, don’t forget to follow our [socials](https://linkin.bio/godot), where we are sure to post some pictures during the event as well.
```

Comment:
Missing link.

Reply:
yes, I'm waiting for the post for updating that

Reply:
Should be up now!